bionic/executor.py

```diff
@@ -50,18 +67,38 @@ class MyManager(SyncManager):
         self._logging_receiver = LoggingReceiver(self._logging_queue)
         self._logging_receiver.start()
 
+        self._process_pool_exec = None
+        self.init_or_resize_process_pool(worker_count)
+
+    def init_or_resize_process_pool(self, worker_count):
+        """
+        Initializes the process pool if not initialized yet and
```

Comment:
nit: this structure is a little weird -- it's not obvious that "if not initialized yet" applies to "returns True", at least until you read the next section. Probably better (and more symmetrical) to start this sentence with "If the process pool is not initialized..."